vulnerabilities/data_dump.py

```diff
@@ -34,15 +34,11 @@ def debian_dump(extract_data, base_release='jessie'):
     Save data scraped from Debian' security tracker.
     """
     for data in extract_data:
-        vulnerability = Vulnerability.objects.create(
-            summary=data.get('description', ''),
-        )
-        VulnerabilityReference.objects.create(
-            vulnerability=vulnerability,
-            reference_id=data.get('vulnerability_id', ''),
+        vulnerability, _ = Vulnerability.objects.get_or_create(
+            cve_id=data['cve_id'],
```

Comment:
What if there is no cve_id in the data? will this crash? And in this case we would likely still want to create a new Vulnerability?

Reply:
This is for the Debian import, where it doesn't look like there will be any entries without CVE ID. At least not in the feed we are currently consuming. The CVE ID is used as a dictionary key:
```
{
  "librsync": {
    "CVE-2014-8242": {
      "scope": "remote",
      "debianbug": 776246,
      "description": "librsync before 1.0.0 uses a truncated MD4 checksum to match blocks",
[...]
```